client/app/components/dynamic-table/index.js

```diff
@@ -12,8 +12,8 @@ function DynamicTable($sanitize) {
   this.orderByReverse = false;
 
   this.pageChanged = () => {
-    const first = this.count * (this.page - 1);
-    const last = this.count * (this.page);
+    const first = this.itemsPerPage * (this.page - 1);
+    const last = this.itemsPerPage * (this.page);
```

Comment:
This issue just happened to me when npm installed Angular 1.6 instead of 1.5. Is it possible you're using Angular 1.6 too?

I'm going to commit npm-shrinkwrap file to the project to avoid such issues in the future.